docs/api/field_type_reference.md

```diff
@@ -1881,15 +1881,19 @@ $validators = [
 
 ## RichText Field Type
 
+!!! caution "Deprecated"
+
+    The RichText Field Type is deprecated since v2.4. Relying on any class from the `eZ\Publish\Core\FieldType` namespace is deprecated.
+
+    If you're implementing any interface or extending any base class from that namespace, refer to its PHPDoc to see what to implement or extend instead.
+    Make sure to enable the eZ Platform RichTextBundle provided by [ezsystems/ezplatform-richtext package](https://github.com/ezsystems/ezplatform-richtext).
```

Comment:
This is ok, to be merged with the starting information mentioned 2 comments above.